packages/app/src/components/TVChart/TVChart.tsx

```diff
@@ -198,7 +199,7 @@ export function TVChart({
 				{ text: '30D', resolution: '1H', description: '30 Days' },
 				{ text: '3M', resolution: '1H', description: '3 Months' },
 			] as TimeFrameItem[],
-			saved_data: chartData,
+			saved_data: isEmpty(chartData) ? undefined : chartData,
```

Comment:
I think it could be better to update `loadChartState ` to return an appropriate type so we don't have deal with the conditional here.

Reply:
My bad for causing this in the first place. Pushed the fix. Nice catch @insulineru and thanks @avclarke.